docs-site/content/guide/high-availability.md

```diff
@@ -272,3 +272,137 @@ final config = Configuration(
+ID                            HOSTNAME   STATUS    AVAILABILITY   MANAGER STATUS   ENGINE VERSION
+z1n71a3h0bw7clclw22i5f0ys     node1      Ready     Active                          20.10.0
+xm1h48xsgzzqftvqaod0nx673     node2      Ready     Active                          20.10.0
+mde8zbj3bsqrvwk02529cm3le     node3      Ready     Active                          20.10.0
+6082x127zz98f0pwgjexbv5xp *   node4      Ready     Active         Leader           20.10.0
+```
+
+- Create a file on the master Swarm node for the docker compose file:
+
+```shell    
+cd typesense && touch docker-stack.yml
+```
+
+- Content for docker-stack.yml
+
+```yaml
+
+version: "3.8"
+services:
+  typesense-1:
+    image: typesense/typesense:0.21.0
+    hostname: typesense-1
+    volumes:
+      - /tmp/typesense-data-1/:/data
+      - ./nodes:/nodes
+    ports:
+      - 8108:8108
+      - 8107:8107
+    command: ["--data-dir", "/data","--api-key", "xyz","--nodes","/nodes","--peering-port","8107","--api-port","8108","--enable-cors"]
+    deploy:
+      replicas: 1
+      labels:
+        feature.description: “typesense-1”
+      restart_policy:
+        condition: any
+      placement:
+        constraints: [node.role == worker]
+
+  typesense-2:
+    image: typesense/typesense:0.21.0
+    hostname: typesense-2
+    volumes:
+      - /tmp/typesense-data-2/:/data
+      - ./nodes:/nodes
+    ports:
+      - 7108:7108
+      - 7107:7107
+    command: ["--data-dir", "/data","--api-key", "xyz","--nodes","/nodes","--peering-port","7107","--api-port","7108","--enable-cors"]
+    deploy:
+      replicas: 1
+      labels:
+        feature.description: “typesense-2”
+      restart_policy:
+        condition: any
+      placement:
+        constraints: [node.role == worker]
+
+  typesense-3:
+    image: typesense/typesense:0.21.0
+    hostname: typesense-3
+    volumes:
+      - /tmp/typesense-data-3/:/data
+      - ./nodes:/nodes
+    ports:
+      - 9108:9108
+      - 9107:9107
+    command: ["--data-dir", "/data","--api-key", "xyz","--nodes","/nodes","--peering-port","9107","--api-port","9108","--enable-cors"]
+    deploy:
+      replicas: 1
+      labels:
+        feature.description: “typesense-3”
+      restart_policy:
+        condition: any
+      placement:
+        constraints: [node.role == worker]
+ ```
+
+- Deploy the stack from Swarm master node
+
+```shell
+docker stack deploy --compose-file docker-stack.yml ts
+```
+
+- List the tasks that are part of the deployed stack
+
+```shell 
+docker stack ps ts
+ID             NAME               IMAGE                        NODE      DESIRED STATE   CURRENT STATE           ERROR     PORTS
+x4w38438c7bn   ts_typesense-1.1   typesense/typesense:0.21.0   node2     Running         Running 5 minutes ago
+0iacq1bhw1ia   ts_typesense-2.1   typesense/typesense:0.21.0   node1     Running         Running 5 minutes ago
+wqyec57a3d4o   ts_typesense-3.1   typesense/typesense:0.21.0   node3     Running         Running 5 minutes ago
+```
```

Comment:
Could we add a set of curl commands to verify that all nodes are up and clustered successfully?

Reply:
yes will include the Debug API call